### PR TITLE
Add content from issues 2 and 3 of Pathfinder Wake the Dead

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/spells/heinous-future.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        "range": {
+            "value": "30 feet"
+        },
+        "rules": [],
+        "save": {
+            "basic": "",
+            "value": "will"
+        },
+        "source": {
+            "value": "Pathfnder: Wake the Dead #3"
```
Comment (simonward) [inline packs/spells/heinous-future.json:44]:
```suggestion
            "value": "Pathfinder: Wake the Dead #3"
```

File: packs/spells/ki-cutting-sight.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        "range": {
+            "value": "30 feet"
+        },
+        "rules": [],
+        "save": {
+            "basic": "",
+            "value": ""
+        },
+        "source": {
+            "value": "Pathfnder: Wake the Dead #2"
```
Comment (simonward) [inline packs/spells/ki-cutting-sight.json:60]:
```suggestion
            "value": "Pathfinder: Wake the Dead #2"
```

File: packs/spells/unseen-heralds.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            "value": ""
+        },
+        "secondarycasters": {
+            "value": "2"
+        },
+        "secondarycheck": {
+            "value": "Arcana or Occultism (whichever isn't used for the primary check), Diplomacy"
+        },
+        "source": {
+            "value": "Pathfnder: Wake the Dead #2"
```
Comment (simonward) [inline packs/spells/unseen-heralds.json:53]:
```suggestion
            "value": "Pathfinder: Wake the Dead #2"
```

File: static/licenses/OpenGameLicense.md [hunk 4]
```diff
@@ -422,6 +422,9 @@ The following text is the property of Wizards of the Coast, Inc. and is Copyrigh
     *  The (Un)Usual Suspects! © 2019, Paizo Inc.; Author: Luis Loza; Illustrators: Miguel Regodón, Emile Denis, Firat Solhan, Yanis Cardin.
 *   Other
     * Pathfinder Special: Fumbus! (Comic Book) © 2021, Paizo Inc.; Authors: Jessica Catalan, Michael Sayre, and Fred Van Lente
+    * Pathfnder: Wake the Dead #1 © 2023, Paizo Inc.; Authors: Mikhail Rekun and Fred Van Lente.
```
Comment (simonward) [inline static/licenses/OpenGameLicense.md:425]:
```suggestion
    * Pathfinder: Wake the Dead #1 © 2023, Paizo Inc.; Authors: Mikhail Rekun and Fred Van Lente.
```

File: static/licenses/OpenGameLicense.md [hunk 5]
```diff
@@ -422,6 +422,9 @@ The following text is the property of Wizards of the Coast, Inc. and is Copyrigh
...
+    * Pathfnder: Wake the Dead #2 © 2023, Paizo Inc.; Authors: Mikhail Rekun and Fred Van Lente.
```
Comment (simonward) [inline static/licenses/OpenGameLicense.md:426]:
```suggestion
    * Pathfinder: Wake the Dead #2 © 2023, Paizo Inc.; Authors: Mikhail Rekun and Fred Van Lente.
```

File: static/licenses/OpenGameLicense.md [hunk 6]
```diff
@@ -422,6 +422,9 @@ The following text is the property of Wizards of the Coast, Inc. and is Copyrigh
...
+    * Pathfnder: Wake the Dead #3 © 2023, Paizo Inc.; Authors: Mikhail Rekun and Fred Van Lente.
```
Comment (simonward) [inline static/licenses/OpenGameLicense.md:427]:
```suggestion
    * Pathfinder: Wake the Dead #3 © 2023, Paizo Inc.; Authors: Mikhail Rekun and Fred Van Lente.
```

File: packs/backgrounds/quick-refugee.json [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                ]
+            }
+        },
+        "description": {
+            "value": "<p>The living residents of Geb are known as the quick, and it is not, overall, a very pleasant thing to be. While some of the quick choose to make the best of their lot, or even thrive in Geb's undead-dominated society, you decided that whatever terrors the outside world holds, they pale in comparison to those that make up the Gebbite upper class. And considering that escape means braving the Mana Wastes, you must have been very, very certain indeed.</p>\n<p>Choose two ability boosts. One must be to <strong>Dexterity</strong> or <strong>Wisdom</strong>, and one is a free ability boost.</p>\n<p>You're trained in Stealth and Undead Lore. When you Hide or @UUID[Compendium.pf2e.actionspf2e.Item.Sneak] successfully, you are also @UUID[Compendium.pf2e.conditionitems.Item.Hidden] or @UUID[Compendium.pf2e.conditionitems.Item.Concealed] from undeads' lifesense and other special precise senses the undead possess.</p>"
+        },
+        "items": {},
+        "rules": [],
+        "source": {
+            "value": "Pathfnder: Wake the Dead #2"
```
[the remark this answers is not in the document]
Comment (TikaelSol) [reply packs/backgrounds/quick-refugee.json:30]: I truly hate the PDF formatting errors that cause this.

File: packs/feats/chaotic-spell.json [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+            "value": "action"
+        },
+        "actions": {
+            "value": 1
+        },
+        "category": "class",
+        "description": {
+            "value": "<p>This is the secret of chaotic magic–your foe can hardly predict and counter your moves if you yourself have no idea what will happen. If the next action you use is to Cast a Spell that deals acid, cold, electricity, fire, or sonic damage, roll a [[/r 1d6]]{d6}. The spell's damage type changes to the type below associated with the result of the roll, and each creature who is damaged by the spell takes the listed additional ef ect.</p>\n<p><strong>1 (Acid):</strong> @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 1}</p>\n<p><strong>2 (Cold):</strong> –10 ft. penalty to all Speeds until the end of your next turn</p>\n<p><strong>3 (Electricity):</strong> @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] until the end of your next turn</p>\n<p><strong>4 (Fire):</strong> Persistent fre damage equal to the spell's level</p>\n<p><strong>5 (Force):</strong> No added efect</p>\n<p><strong>6 (Sonic):</strong>@UUID[Compendium.pf2e.conditionitems.Item.Deafened] for 2 rounds</p>"
```
Comment (simonward) [inline packs/feats/chaotic-spell.json:14]:
```suggestion
            "value": "<p>This is the secret of chaotic magic–your foe can hardly predict and counter your moves if you yourself have no idea what will happen. If the next action you use is to Cast a Spell that deals acid, cold, electricity, fire, or sonic damage, roll a [[/r 1d6]]{d6}. The spell's damage type changes to the type below associated with the result of the roll, and each creature who is damaged by the spell takes the listed additional effect.</p>\n<p><strong>1 (Acid):</strong> @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 1}</p>\n<p><strong>2 (Cold):</strong> –10 ft. penalty to all Speeds until the end of your next turn</p>\n<p><strong>3 (Electricity):</strong> @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] until the end of your next turn</p>\n<p><strong>4 (Fire):</strong> Persistent fire damage equal to the spell's level</p>\n<p><strong>5 (Force):</strong> No added efect</p>\n<p><strong>6 (Sonic):</strong>@UUID[Compendium.pf2e.conditionitems.Item.Deafened] for 2 rounds</p>"
```

Comment (simonward) [reply packs/feats/chaotic-spell.json:14]: Could also make the persistent fire damage into a scaling damage link.

**Review by simonward — APPROVED** [review]

**Pull request merged**